examples/ruby/sinatra/gift_giver/lib/attendee_refresher.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class AttendeeRefresher
+  class << self
+    def call
+      HTTParty
+        .get(url)
+        .each do |attendee|
+          member = attendee['member']
+
+          Attendee.find_or_create_by!(
+            name: member['name'],
```

Comment:
Can't find by name.
Two users may have the same name.

Please, use only `vendor_user_id` as key.

Reply:
Optionally it can update the name once find by id

Reply:
Cool, so we can use:
```ruby
attendee = Attendee.find_or_initialize_by(
  vendor_user_id: member['vendor_user_id']
)
attendee.name = member['name']
attendee.save!
```